node/node.go

```diff
@@ -870,6 +901,21 @@ func (n *Node) GetAccount(address string, options api.AccountQueryOptions) (api.
 	}, blockInfo, nil
 }
 
+func adaptBlockInfoIfError(err error) (api.BlockInfo, error) {
+	if err == nil {
+		return api.BlockInfo{}, nil
+	}
+
+	apiBlockInfo, ok := extractApiBlockInfoIfErrAccountNotFoundAtBlock(err)
+	if ok {
+		return apiBlockInfo, nil
+	}
+	if err == ErrCannotCastAccountHandlerToUserAccountHandler {
```

Comment:
A comment explaining why this isn't an actual error would be helpful for posterity.

Reply:
added